github-management/default-branch-migration.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Default Branch Migration
+linkTitle: Default Branch Migration
+description:  |
+  Instructions on how to migrate the default branch from master to main.
+weight: 99
+type: docs
+aliases: [ "/rename" ]
+---
+
+# Default Branch Migration
+
+This document outlines steps needed to migrate the default branch
+of your repo from `master` to `main`.
+
+Note: This document is currently a work in progress.
+
+If you have questions about the process, reach out to the GitHub Management Team
+on the [#github-management] channel on slack or open an issue in the [kubernetes/org] repo.
+
+## Prerequisites
+
+- [ ] Ensure that your repo doesn't have too many:
+  - [ ] open PRs. The branch rename will re-trigger prow on _all_ PRs,
+    which will cause a huge spike in the CI load.
+  - [ ] periodic jobs. Changes to frequent periodics haven't been tested yet.
```

Comment:
I don't have a number for this. 20 again? Or maybe 10?

How many people are going to get how many alert emails of these jobs fail? If it's not cross-sig, you are probably fine.

But like kops for example, I wouldn't want all of those jobs failing at once and hammering people's inboxes.